CMakeLists.txt

```diff
@@ -451,20 +453,23 @@ if (WIN32)
   set (CMAKE_RUNTIME_OUTPUT_DIRECTORY "${PROJECT_BINARY_DIR}/bin")
 endif ()
 
-# The list of tools (to be installed into, e.g., /usr/local/bin)
-set (TOOLS CartConvert ConicProj GeodesicProj GeoConvert GeodSolve
-  GeoidEval Gravity IntersectTool MagneticField Planimeter RhumbSolve
-  TransverseMercatorProj)
+if (BUILD_TOOLS)
+  # The list of tools (to be installed into, e.g., /usr/local/bin)
+  set (TOOLS CartConvert ConicProj GeodesicProj GeoConvert GeodSolve
+    GeoidEval Gravity IntersectTool MagneticField Planimeter RhumbSolve
+    TransverseMercatorProj)
+
+  add_subdirectory (tools)
+endif()
+
 # The list of scripts (to be installed into, e.g., /usr/local/sbin)
 set (SCRIPTS geographiclib-get-geoids geographiclib-get-gravity
```

Comment:
the `SCRIPTS` is used by tools/CMakeLists.txt and man/CMakeLists.txt, so excluding it too may break man.